### ISSUE TITLE
typeorm@0.2.41 Support

### ISSUE THREAD

**Issue by cmd2001** [devniel/nestjs-typeorm-testing#15]
This package seems not compatible with newest typeorm, when trying to use it with `typeorm@0.2.41`, it gets `TypeError: cb is not a function`.

How to reproduce this issue?

Just run `npm install typeorm@0.2.41`, then run `npm run test`, you will get
```plain
 FAIL  src/tests/testing-module.spec.ts
  ● Create multiple testing modules › should create multiple testing modules when using different names
    TypeError: cb is not a function
      18 | export const fakeConnection = {
      19 |   query: (query, parameters, cb) => {
    > 20 |     cb(null, {
         |     ^
      21 |       rows: 0,
      22 |     });
      23 |   },
      at Object.query (lib/createFakeConnection.ts:20:5)
      at driver/postgres/PostgresDriver.ts:1206:24
      at FakeDBDriver.Object.<anonymous>.PostgresDriver.executeQuery (driver/postgres/PostgresDriver.ts:1205:16)
      at FakeDBDriver.<anonymous> (driver/postgres/PostgresDriver.ts:359:36)
      at step (../node_modules/typeorm/node_modules/tslib/tslib.js:143:27)
      at Object.next (../node_modules/typeorm/node_modules/tslib/tslib.js:124:57)
      at fulfilled (../node_modules/typeorm/node_modules/tslib/tslib.js:114:62)

```

**Comment by devniel** [comment]
It's because of this change, https://github.com/typeorm/typeorm/commit/1197f881382d5d7a1f604ab53ae34e9461bf1ffe

change is in progress.

**Comment by devniel** [comment]
Solved with version `0.1.4-alpha`; version `0.1.5.-alpha` also supports latest `@nestjs/typeorm`.
